Documentation/running-tests.md

```diff
@@ -98,6 +98,21 @@ requires that the kernel be compiled with `CONFIG_BLK_DEV_ZONED` enabled.
 RUN_ZONED_TESTS=1
 ```
 
+### Running nvme-rdma nvmeof-mp srp tests
+
+These tests default will use rdma_rxe driver, and it also support use siw driver.
```

Comment:
How about changing this sentence into the following:

Most of these tests will use the rdma_rxe (soft-RoCE) driver by default. The siw (soft-iWARP) driver is also supported.

Reply:
looks more reasonable, just updated as you suggested, thanks.